hub/cli/auth.py

```diff
@@ -5,6 +5,7 @@
 """
 
 import click
+import hub
```

Comment:
This `import hub` is still here, though.

Reply:
removed